romanisim/l3.py

```diff
@@ -1,14 +1,33 @@
-"""Function for Level 3-like images.
+"""Roman WFI simulator functions for Level 3 mosaics.
 
+Based on galsim's implementation of Roman image simulation.  Uses galsim Roman modules
+for most of the real work.
 """
 
 import numpy as np
+import math
+import astropy.time
+from astropy import table
 import galsim
-from . import log
-from . import image, wcs, psf
+from galsim import roman
 
+from . import parameters
+from . import util
+import romanisim.wcs
+import romanisim.l1
+import romanisim.bandpass
+import romanisim.psf
+import romanisim.image
+import romanisim.persistence
+from romanisim import log
+import roman_datamodels.maker_utils as maker_utils
 
-def add_objects_to_l3(l3_mos, source_cat, rng=None, seed=None):
+# Define centermost SCA for PSFs
+CENTER_SCA = 2
```

Comment:
Let's probably put this in romanisim.parameters, possibly renaming it to default_sca as there isn't a good center.

Reply:
Changed